gc/gc-base/src/main/java/org/projectnessie/gc/expire/local/DefaultLocalExpire.java

```diff
@@ -67,6 +67,12 @@ public DeleteSummary expire() {
     Instant started = clock().instant();
     expireParameters().liveContentSet().startExpireContents(started);
 
+    if (parallelism() == 1) {
```

Comment:
Looks like this can be unified with the try-finally below, primarily to not omit the logging.

Reply:
Done